packages/gestalt/src/Masonry.js

```diff
@@ -186,6 +186,10 @@ export default class Masonry<T> extends React.Component<Props<T>, State<T>> {
     virtualize: false,
   };
 
+  gridWrapper: { current: HTMLElement | null } = React.createRef();
```

Comment:
should we clean this up to be a `HTMLDivElement` since we are here?

Reply:
We don't actually care if it's a `div` or `span` or whatever in this case, as long as it exposes a `clientWidth`. I'd rather pick the most ambiguous type that still accomplishes 100% type safety.

Reply:
Hmm, Id probably pick the most specific type that still covers all cases but non-blocker